.github/workflows/format-lint-test.yaml

```diff
@@ -0,0 +1,31 @@
+name: Format, Lint, Test
+
+on:
+  push:
+    branches: [ "dev" ]
```

Comment:
Pushing on dev is not possible right so do we really need this?

On the other hand, if some reason someone does actually push to dev it might be nice to make sure it is at least formatted :p

Reply:
Yeah, I know it's somewhat redundant to run _again_ after merge to `dev`, so I can also remove it if you like. It's just another assertion that the current `dev` always works.

Reply:
I don't think though this will trigger once a branch is merged into `dev` though. I think it is only triggered when someone directly pushes their commit from their local `dev` branch to remote. I can be mistaken though.

Reply:
But we should not do that anyway, right? I will just leave it out. 👍

Reply:
Reintroduced it 😄